beacon_node/lighthouse_network/src/rpc/codec.rs

```diff
@@ -1097,21 +1090,24 @@ mod tests {
         }
     }
 
-    fn bbroot_request_v1(spec: &ChainSpec) -> BlocksByRootRequest {
-        BlocksByRootRequest::new_v1(vec![Hash256::zero()], spec)
+    fn bbroot_request_v1(fork_name: ForkName) -> BlocksByRootRequest {
+        let fork_context = fork_context(fork_name);
```

Comment:
nit:
same I'd inline all the `fork_context()` calls for cleaner diff and less lines of code